[@mantine/core] Tooltip: Add the `radius` prop

jerebtw opened the pull request:
Fixes https://github.com/mantinedev/mantine/issues/560

rtivital commented:
How does it solve the bug with arrow discussed in the issue?

jerebtw commented:
We still need to find a fix for that problem

rtivital commented:
I do not think that we should provide a feature that does not work, so this PR can be merged only with a fix

jerebtw commented:
Okay I will think about a fix

jerebtw commented on src/mantine-core/src/components/Popper/Popper.tsx:
```diff
@@ -21,6 +21,9 @@ export interface SharedPopperProps {
   /** Arrow size in px */
   arrowSize?: number;
 
+  /** Arrow distance to the left/right * arrowSize */
```
@rtivital is that comment right?

jerebtw commented:
> I do not think that we should provide a feature that does not work, so this PR can be merged only with a fix

I fixed the issue 👍🏻 I just moved the arrow a little

rtivital commented:
Thanks, it works great!
